bundles/org.eclipse.ui.editors/META-INF/MANIFEST.MF

```diff
@@ -15,7 +15,8 @@ Export-Package:
  org.eclipse.ui.internal.editors.text.codemining.annotation;x-internal:=true,
  org.eclipse.ui.internal.texteditor;x-internal:=true,
  org.eclipse.ui.internal.texteditor.stickyscroll;x-internal:=true,
- org.eclipse.ui.texteditor
+ org.eclipse.ui.texteditor,
+ org.eclipse.ui.texteditor.stickyscroll;x-internal:=true
```

Comment:
so we would remove the "x-internal:=true" part to make this "real" API once we are confident enough?

Reply:
Yes, that would be the idea. But I'm not sure if this is the right approach or if this is a good idea.